app/mailers/embargo_mailer.rb

```diff
@@ -0,0 +1,64 @@
+# -*- encoding : utf-8 -*-
+# models/embargo_mailer.rb:
+# Alerts relating to embargoes.
+#
+# Copyright (c) 2017 UK Citizens Online Democracy. All rights reserved.
+# Email: hello@mysociety.org; WWW: http://www.mysociety.org/
+
+class EmbargoMailer < ApplicationMailer
+  def self.alert_expiring
+    InfoRequest.embargo_expiring.group_by(&:user).each do |user, info_requests|
+      info_requests.reject! do |info_request|
+        alert_event_id = info_request.last_event_forming_initial_request.id
```

Comment:
Aha, ok, I think I misunderstood the event ID as not being important (hence just putting something in the field) because I didn't think about extensions to the embargo then expiring again.

It feels like an "embargo extended" event is really what we're after (as it might actually be a useful thing to print in a timeline for example), but we already save the extensions in a separate table, so perhaps for this purpose we could just use their ids instead of an event? (Perhaps with a special 0 id, for the initial embargo or something?).